collector/osv/src/server.rs

```diff
@@ -260,29 +255,29 @@ pub async fn collect_packages(
                                     }
                                 }
 
-                                for cvss_v3 in &cvss_v3s {
-                                    if let Err(err) = state
-                                        .guac_client
-                                        .intrinsic()
-                                        .ingest_vuln_metadata(
-                                            &vulnerability_input_spec,
-                                            &VulnerabilityMetadataInputSpec {
-                                                score_type: if cvss_v3.minor_version == 0 {
-                                                    VulnerabilityScoreType::CVSSv3
-                                                } else {
-                                                    VulnerabilityScoreType::CVSSv31
-                                                },
-                                                score_value: cvss_v3.score().value(),
-                                                timestamp: Default::default(),
-                                                origin: "osv".to_string(),
-                                                collector: "osv".to_string(),
-                                            },
-                                        )
-                                        .await
-                                    {
-                                        collected_guac_errors.push(err)
-                                    }
-                                }
+                                // for cvss_v3 in &cvss_v3s {
```

Comment:
Should we remove this for good?

Reply:
I was keen to, at least, try to add the vulnerability metadata tables to guac and then de-comment this block.